code/drasil-lang/Language/Drasil/Chunk/Eq.hs

```diff
@@ -94,31 +94,33 @@ fromEqnSt' nm desc def symb sp =
   EC (dqd' (cc' (mkIdea nm desc Nothing) def) symb sp Nothing) []
 
 -- | Wrapper for fromEqnSt and fromEqnSt'
-mkQDefSt :: UID -> NP -> Sentence -> (Stage -> Symbol) -> Space ->
+mkQDefSt :: String -> NP -> Sentence -> (Stage -> Symbol) -> Space ->
   Maybe UnitDefn -> Expr -> QDefinition
 mkQDefSt u n s symb sp (Just ud) e = fromEqnSt u n s symb sp ud e
 mkQDefSt u n s symb sp Nothing   e = fromEqnSt' u n s symb sp e
 
 -- | Used to help make 'QDefinition's when 'UID', term, and 'Symbol' come from the same source.
 mkQuantDef :: (Quantity c, MayHaveUnit c) => c -> Expr -> QDefinition
-mkQuantDef c = mkQDefSt (c ^. uid) (c ^. term) EmptyS (symbol c) (c ^. typ) (getUnit c)
+mkQuantDef c = mkQDefSt (uidToStr $ c ^. uid) (c ^. term) EmptyS (symbol c) (c ^. typ) (getUnit c)
```

Comment:
This back-and-forth says that there should be a `mkQDefSt'` that takes a `UID` directly. This might propagate to needing other things below to support that.

Reply:
Ahh okay, I think we can actually just make `mkWDefSt` take a `UID` since its only used with UIDs anyway

Reply:
Prefect.